misc/ballerina-bindgen/src/main/java/org/ballerinalang/bindgen/command/BindingsGenerator.java

```diff
@@ -79,6 +88,27 @@
     private static Map<String, String> failedClassGens = new HashMap<>();
 
     void generateJavaBindings() throws BindgenException {
+        if (projectRoot != null) {
+            Manifest manifest = TomlParserUtils.getManifest(projectRoot);
+            if (manifest != null) {
+                List<Library> platformLibraries = manifest.getPlatform().getLibraries();
+                if (platformLibraries != null) {
+                    for (Library library: platformLibraries) {
+                        if (library.path != null) {
+                            classPaths.add(Paths.get(projectRoot.toString(), library.path).toString());
+                        } else if (library.groupId != null && library.artifactId != null && library.version != null) {
+                            mavenResolver(library.groupId, library.artifactId, library.version, projectRoot);
+                        }
+                    }
+                }
+            }
+        }
```

Comment:
Isn't it better to move this logic to a separate method? So we can reduce the complexity of the `generateJavaBindings` method.

Reply:
Done